src/App.tsx

```diff
@@ -49,6 +50,7 @@ const ApolloApp = () => {
             <Error />
             <WalletSelector />
             <MintPopup />
+            <DicePopup />
```

Comment:
Don't put popup in App. I put all popup in page now https://github.com/OtterClam/otto-app/blob/main/src/pages/store/StorePage.tsx#L145:L145

Reply:
done

Reply:
done